msm/src/circuit_design/constraints.rs

```diff
@@ -40,9 +40,11 @@ impl<F: PrimeField, CIx: ColumnIndexer, LT: LookupTableID> ColAccessCap<F, CIx>
     fn assert_zero(&mut self, cst: Self::Variable) {
         // FIXME: We should enforce that we add the same expression
```

Comment:
This comment can also be removed. The comment was regarding the constrain_index.